Realm/ObjectServerTests/SwiftObjectServerTests.swift

```diff
@@ -2931,4 +2931,45 @@ class CombineObjectServerTests: SwiftSyncTestCase {
     }
 }
 
+@available(macOS 12.0, *)
+class AsyncAwaitObjectServerTests: SwiftSyncTestCase {
+    func testAsyncOpenStandalone() async throws {
+        try autoreleasepool {
+            let realm = try Realm()
+            try! realm.write {
+                (0..<10000).forEach { _ in realm.add(SwiftPerson(firstName: "Charlie", lastName: "Bucket")) }
```

Comment:
Adding this many objects seems like it'd just make the test slow without making it any better at catching bugs.